tests/unit/acquisition/function/test_active_learning.py

```diff
@@ -235,3 +244,151 @@ def test_bichon_ranjan_criterion(threshold: float, at: tf.Tensor, alpha: float,
     )
 
     npt.assert_allclose(actual, expected, rtol=0.01)
+
+
+def test_integrated_variance_reduction() -> None:
+
+    x = to_default_float(tf.constant(np.arange(1, 7).reshape(-1, 1) / 8.0))  # shape: [6, 1]
+    y = fnc_2sin_x_over_3(x)
+
+    model6 = GaussianProcessRegression(gpr_model(x, y))
+    model5 = GaussianProcessRegression(gpr_model(x[:5, :], y[:5, :]))
+    reduced_data = Dataset(x[:5, :], y[:5, :])
+    query_points = x[5:, :]
+    integration_points = tf.concat([0.37 * x, 1.7 * x], 0)  # shape: [14, 1]
+
+    _, pred_var6 = model6.predict(integration_points)
+
+    acq_noweight = IntegratedVarianceReduction(integration_points=integration_points)
+    acq = IntegratedVarianceReduction(threshold=[0.5, 0.8], integration_points=integration_points)
+
+    acq_function = acq.prepare_acquisition_function(model=model5, dataset=reduced_data)
+    acq_function_noweight = acq_noweight.prepare_acquisition_function(
+        model=model5, dataset=reduced_data
+    )
+    acq_values = -acq_function(tf.expand_dims(query_points, axis=-2))
```

Comment:
why minus here?

Reply:
acquisition function returns minus new variance, I flip it so that everythin is a variance